presto-docs/src/main/sphinx/release/release-307.rst

```diff
@@ -0,0 +1,93 @@
+===========
+Release 307
+===========
+
+General Changes
+---------------
+
+* Fix cleanup of spill files for queries using window functions or ``ORDER BY``. (:issue:`543`)
+* Optimize queries containing ``ORDER BY`` together with ``LIMIT`` over an ``OUTER JOIN``
+  by pushing ``ORDER BY`` and ``LIMIT`` to the outer side of the join. (:issue:`419`)
+* Improve performance of table scans for data sources that produce tiny pages. (:issue:`467`)
+* Improve performance of ``IN`` subquery expressions by removing distinct aggregations. (:issue:`551`)
```

Comment:
This should be “Improve performance of IN subquery expressions that contain a DISTINCT clause”